test_runner/src/main/kotlin/ftl/client/google/run/android/GcAndroidTestMatrix.kt

```diff
@@ -58,8 +58,14 @@ private fun createAndroidTestMatrix(
     runIndex: Int
 ): Testing.Projects.TestMatrices.Create {
 
+    val clientDetails = if (testMatrixType is TestMatrixAndroid.Type.Instrumentation && testMatrixType.clientDetails.isNotEmpty()) {
+        ClientInfo()
+            .setName("Flank")
+            .setClientInfoDetails(testMatrixType.clientDetails.toClientInfoDetailList())
```

Comment:
hey, we have some extension for `ClientInfo` [here](https://github.com/Flank/flank/blob/a9232d755d04b5d1646d0f496f67be48539ba843/test_runner/src/main/kotlin/ftl/client/google/run/Utils.kt#L24) could you reuse it?